jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPartFormData.java

```diff
@@ -450,22 +513,21 @@ private class PartsListener extends MultiPart.AbstractPartsListener
             public void onPartContent(Content.Chunk chunk)
             {
                 ByteBuffer buffer = chunk.getByteBuffer();
+                long maxFileSize = getMaxFileSize();
+                size += buffer.remaining();
+                if (maxFileSize >= 0 && size > maxFileSize)
```

Comment:
Can we rename, at least internally?
```suggestion
                if (maxPartSize >= 0 && size > maxPartSize)
```